packages/theme/src/color-ramps/lib/index.ts

```diff
@@ -166,31 +166,21 @@ function calculateRamp( {
 			adjustedTarget,
 			computedDir,
 			{
-				strict: false,
 				lightnessConstraint,
 				taperChromaOptions,
 			}
 		);
 
 		// When the target contrast is not met, take note of it and use
 		// that information to guide the ramp calculation bisection.
-		if ( ! searchResults.reached && ! contrast.ignoreWhenAdjustingSeed ) {
-			SATISFIED_ALL_CONTRAST_REQUIREMENTS = false;
-
-			// Calculate constraint failure severity for seed optimization
-			// Use the relative deficit size, weighted by how changing the seed would impact this constraint
-			const deficitVsTarget = adjustedTarget - searchResults.achieved;
-
-			// Weight the deficit by how much seed adjustment would help this constraint
-			// If seed has low contrast vs reference, adjusting seed has high impact
-			// If seed has high contrast vs reference, adjusting seed has low impact
-			const impactWeight = 1 / getContrast( seed, referenceColor );
-			const weightedDeficit = deficitVsTarget * impactWeight;
```

Comment:
This code, weighing the contrast by dividing by the seed-vs-reference contrast, seems wrong to me.

First, seed adjustment helps all constraints equally. If the seed luminance (Y) is halved, the contrast between reference and target approximately doubles. This is universal, no matter where the intermediate "reference" value is.

The right way to measure the deficit in "same units" is to do the opposite: multiply the deficit with seed-vs-reference contrast instead of dividing it. Then the result is always "contrast between seed and target", independent from the intermediate steps.